packages/cookie-consent/cookie-consent.scss

```diff
@@ -0,0 +1,88 @@
+@import "~@fremtind/jkl-core/variables/all";
+@import "~@fremtind/jkl-core/mixins/all";
+@import "~@fremtind/jkl-button/button.scss";
+@import "~@fremtind/jkl-list/list.scss";
+
+$card-corner-radius: rem(2px);
+$container-width: rem(660px);
+$container-width--compact: rem(320px);
+
+.jkl-cookie-consent-modal + backdrop {
+    position: fixed;
+    background: rgba(0, 0, 0, 0.3);
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    transition: 0.2s;
+    z-index: $z-index--modal !important;
+}
+.jkl-cookie-consent-modal[hidden],
+.jkl-cookie-consent-modal + backdrop[hidden] {
+    pointer-events: none;
+    visibility: hidden;
+    display: block;
+    opacity: 0;
+}
+
+.jkl-cookie-consent-modal {
+    z-index: $z-index--floating !important;
+    position: absolute;
+    margin: auto;
+    top: 15%;
+    left: 0;
+    right: 0;
+    width: 100%;
+    padding: $layout-spacing--medium $layout-spacing--medium rem(30px) $layout-spacing--medium;
+    min-width: $container-width--compact;
+    max-width: $container-width;
+    background: $hvit;
+    box-shadow: 0px 4px 20px rgba(0, 0, 0, 0.25);
```

Comment:
Kunne kanskje brukt en variabel her, typ $sort. Også kunne man brukt den innebygde funksjonen transparentize

https://sass-lang.com/documentation/modules/color#transparentize

Reply:
Sånn egentlig er vel det mest riktig å bruke skygge-variablene fra core. Oppdaterer med det 😃

Reply:
Hehe, de (den?) skyggevariablene er kanskje ikke helt up to date heller 😅 Men er fint å bruke dem uansett!